pkg/api/v1/client.go

```diff
@@ -79,6 +82,11 @@ func (c *Client) SetToken(token string) {
 	c.authToken = token
 }
 
+// SetDumper sets requests and responses to be written to the given writer for debugging.
+func (c *Client) SetDumper(w io.Writer) {
+	c.dumper = w
+}
```

Comment:
uhm... the client already wraps the http client with the `Doer` interface in this file at line 23. Why not write a wrapper around that instead of adding direct calls to the dumper?

Reply:
apologies, I haven't responded here sooner - I'll be testing this suggestion out to see if it works as expected.